department-of-veterans-affairs/caseflow
# Show appropriate error when UI receives server error

Connects #6485 

### Description
Pass appropriate error messages to LoadingDataDisplay component so that the UI renders an appropriate error message to the user.

### Acceptance Criteria 

### Testing Plan
1. Upload an RO or Judge file.
2. Get to the point of confirm
3. Terminate the VACOLS_Development docker instance using the 'docker stop' command
4. Press the big red button
5. Verify the error screen shows up with the error message described in the ticket.
